deepchem/molnet/run_benchmark.py

```diff
@@ -209,3 +209,91 @@ def run_benchmark(datasets,
         output_line.extend(
             ['time_for_running', time_finish_fitting - time_start_fitting])
         writer.writerow(output_line)
+
+
+#
+# Note by @XericZephyr. Reason why I spun off this function:
+#   1. Some model needs dataset information.
+#   2. It offers us possibility to **cache** the dataset
+#      if the featurizer runs very slow, e.g., GraphConv.
+#   2+. The cache can even happen at Travis CI to accelerate
+#       CI testing.
+#
+def load_dataset(dataset, featurizer, split=None):
+  """
```

Comment:
@XericZephyr Sorry, did not notice this. Default value of `split` may not be set as `None` since dataset loading functions do not have any splitter corresponding to `None`.
There are two alternatives:
1) change the default to 'random', which is universally accepted
2) if `split` is not received, do not input it into the dataset loading functions
I could help to do the quick change, what's your preference?